### PR TITLE
Improve check integrity, solve problematic countries request, move cursor to the init

### PR DESCRIPTION
**Pull request by InbarShirizly**
image

#14 , #12

### CODE REVIEW

**Comment by InbarShirizly** [comment]
**lines of code to be aware:** 

https://github.com/nirbarazida/Data-mining-project/blob/2b19248cd0d10aee09eb7684a6af93fdc91d66ab/src/ORM.py#L110
https://github.com/nirbarazida/Data-mining-project/blob/2b19248cd0d10aee09eb7684a6af93fdc91d66ab/src/ORM.py#L126

File: src/user.py [hunk 1]
```diff
@@ -50,19 +50,21 @@ def reputation(self):
...
-
-            # insert user information to not scrapped users logger, Can't print user name with unknown characters
-            logger.error(f'IntegrityError: "Duplicate entry for key users.name ranked {self._rank} at'
-                                      f' website {self._website_name} with url: {self._url}" ')
+            # # log the problem in user logger, Can't print user name with spacial/ unknown characters
+            # logger.warning(f'IntegrityError: "Duplicate entry for key users.name ranked')
+            #
+            # # insert user information to not scrapped users logger, Can't print user name with unknown characters
+            # logger.error(f'IntegrityError: "Duplicate entry for key users.name ranked {self._rank} at'
+            #                           f' website {self._website_name} with url: {self._url}" ')
```
Comment (nirbarazida) [inline src/user.py:67]: Are we also logging the error?
next time when having an issue we would like to know what caused it

Comment (InbarShirizly) [reply src/user.py:67]: Yes, we do it in  line 60 -maybe you can modify the logger to print the error and some adjusted string

Comment (InbarShirizly) [reply src/user.py:67]: Yes, we do it in line 60 -maybe you can modify the logger to print the error and some adjusted string

https://github.com/nirbarazida/Data-mining-project/blob/2b19248cd0d10aee09eb7684a6af93fdc91d66ab/src/user.py#L60

**Pull request merged**